package/MDAnalysis/analysis/base.py

```diff
@@ -27,6 +27,8 @@
 
 import numpy as np
 import logging
+from MDAnalysis.lib.log import ProgressMeter
```

Comment:
Thanks for the help! On it.

On Tue, May 24, 2016 at 8:55 AM kain88-de <notifications@github.com> wrote:

> In package/MDAnalysis/analysis/base.py
> <https://github.com/MDAnalysis/mdanalysis/pull/856#discussion_r64420133>:
>
> > @@ -27,6 +27,8 @@
> >
> >  import numpy as np
> >  import logging
> > +from MDAnalysis.lib.log import ProgressMeter
>
> This import doesn't seem to be used anymore.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/MDAnalysis/mdanalysis/pull/856/files/c41cf9540291e37191f253bfde996a7e0a6f3767#r64420133>
>
-- 
Have a wonderful day,
John J. Detlefs